<?xml version='1.0' encoding='utf-8'?>
<ui version="4.0">
 <class>AnnotationControlWidget</class>
 <widget class="QWidget" name="AnnotationControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>1102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Annotation Control</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBoxFunctionType">
     <property name="title">
      <string>機能タイプ</string>
     </property>
     <layout class="QGridLayout" name="gridLayoutFunctionTypes">
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxCaption">
        <property name="text">
         <string>Caption生成</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox {
    font-size: 10px;
    font-weight: normal;
    spacing: 5px;
}
QCheckBox::indicator {
    width: 14px;
    height: 14px;
}
QCheckBox::indicator:unchecked {
    border: 1px solid #ccc;
    background-color: white;
    border-radius: 2px;
}
QCheckBox::indicator:checked {
    border: 1px solid #4CAF50;
    background-color: #4CAF50;
    border-radius: 2px;
}</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxTagger">
        <property name="text">
         <string>Tag生成</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox {
    font-size: 10px;
    font-weight: normal;
    spacing: 5px;
}
QCheckBox::indicator {
    width: 14px;
    height: 14px;
}
QCheckBox::indicator:unchecked {
    border: 1px solid #ccc;
    background-color: white;
    border-radius: 2px;
}
QCheckBox::indicator:checked {
    border: 1px solid #4CAF50;
    background-color: #4CAF50;
    border-radius: 2px;
}</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBoxScorer">
        <property name="text">
         <string>品質スコア</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox {
    font-size: 10px;
    font-weight: normal;
    spacing: 5px;
}
QCheckBox::indicator {
    width: 14px;
    height: 14px;
}
QCheckBox::indicator:unchecked {
    border: 1px solid #ccc;
    background-color: white;
    border-radius: 2px;
}
QCheckBox::indicator:checked {
    border: 1px solid #4CAF50;
    background-color: #4CAF50;
    border-radius: 2px;
}</string>
        </property>
       </widget>
      </item>
     </layout>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxProviderSelection">
     <property name="title">
      <string>実行環境選択</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayoutProviders">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBoxWebAPI">
        <property name="text">
         <string>Web API</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox {
    font-size: 10px;
    font-weight: normal;
    spacing: 5px;
}
QCheckBox::indicator {
    width: 14px;
    height: 14px;
}
QCheckBox::indicator:unchecked {
    border: 1px solid #ccc;
    background-color: white;
    border-radius: 2px;
}
QCheckBox::indicator:checked {
    border: 1px solid #4CAF50;
    background-color: #4CAF50;
    border-radius: 2px;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxLocal">
        <property name="text">
         <string>ローカルモデル</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox {
    font-size: 10px;
    font-weight: normal;
    spacing: 5px;
}
QCheckBox::indicator {
    width: 14px;
    height: 14px;
}
QCheckBox::indicator:unchecked {
    border: 1px solid #ccc;
    background-color: white;
    border-radius: 2px;
}
QCheckBox::indicator:checked {
    border: 1px solid #4CAF50;
    background-color: #4CAF50;
    border-radius: 2px;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacerProviders">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxModelSelection">
     <property name="title">
      <string>モデル選択</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutModels">
      <item>
       <widget class="ModelSelectionTableWidget" name="modelSelectionTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxOptions">
     <property name="title">
      <string>オプション</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutOptions">
      <item>
       <widget class="QCheckBox" name="checkBoxLowResolution">
        <property name="text">
         <string>API負荷軽減用512x512ベースの画像を使用</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox {
    font-size: 10px;
    font-weight: normal;
    spacing: 5px;
}
QCheckBox::indicator {
    width: 14px;
    height: 14px;
}
QCheckBox::indicator:unchecked {
    border: 1px solid #ccc;
    background-color: white;
    border-radius: 2px;
}
QCheckBox::indicator:checked {
    border: 1px solid #4CAF50;
    background-color: #4CAF50;
    border-radius: 2px;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxBatchMode">
        <property name="text">
         <string>バッチ処理モード</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox {
    font-size: 10px;
    font-weight: normal;
    spacing: 5px;
}
QCheckBox::indicator {
    width: 14px;
    height: 14px;
}
QCheckBox::indicator:unchecked {
    border: 1px solid #ccc;
    background-color: white;
    border-radius: 2px;
}
QCheckBox::indicator:checked {
    border: 1px solid #4CAF50;
    background-color: #4CAF50;
    border-radius: 2px;
}</string>
        </property>
       </widget>
      </item>
     </layout>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutControls">
     <item>
      <widget class="QPushButton" name="pushButtonStart">
       <property name="toolTip">
        <string>選択したモデルでアノテーション実行</string>
       </property>
       <property name="text">
        <string>実行</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    font-size: 12px;
    font-weight: bold;
    padding: 8px 16px;
    border: 2px solid #4CAF50;
    border-radius: 6px;
    background-color: palette(button);
    color: palette(buttonText);
    min-height: 30px;
}
QPushButton:hover {
    background-color: palette(highlight);
    color: palette(highlightedText);
}
QPushButton:pressed {
    background-color: #4CAF50;
    color: white;
}
QPushButton:disabled {
    background-color: palette(window);
    color: palette(mid);
    border-color: palette(mid);
}</string>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ModelSelectionTableWidget</class>
   <extends>QWidget</extends>
   <header>lorairo.gui.widgets.model_selection_table_widget</header>
  </customwidget>
 </customwidgets>
 <resources />
 <connections />
</ui>
